src/lib/components/TermsModal.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  import { getModalStore } from '@skeletonlabs/skeleton';
+  const modalStore = getModalStore();
+
+  // Ideally, this should be a value from the database that has timestamps, author, etc.
+  // For now, the easiest place to find and update the html used here is in the src file.
+  import Terms from '../../terms.svelte';
```

Comment:
I think this should also be `$lib`?

Reply:
I put it in the src folder because I thought it would be easier to find and swap with jenkins. I can move it to the lib folder.

Reply:
oh, no that is fine, just checking